packages/fxa-auth-server/bin/key_server.js

```diff
@@ -59,6 +60,13 @@ async function run(config) {
     Container.set(AuthFirestore, authFirestore);
   }
 
+  const accountEventsManager = config.accountEvents.enabled
+    ? new AccountEventsManager()
+    : {
+        recordEmailEvent: async () => Promise.resolve(),
+      };
```

Comment:
👍   Every time I see DI Container being used I'm a little worried about Container.get called on a dep somewhere without a Container.set call somewhere up the dependency graph.

We _might_ need to set AccountEventsManager in the container for some of the remote tests.  🤷

Reply:
Good point, it seems like it is typically our scripts that get this error. I think I got all the remote tests 😅